test/integration_selenium/test_toolbox_filters.py

```diff
@@ -26,6 +26,8 @@ def test_toolbox_filters(self):
         the specified section is no longer displayed in the browser.
         """
         self.register()
+        self.sleep_for(self.wait_types.UX_RENDER)
```

Comment:
```suggestion
```
This is unlikely to fix the test on its own if the next step waits for a specific element (waiting for an element waits up to the default timeout, which is 10 seconds, while UX_RENDER is 1 second).

Reply:
That's a good point, I think I need to move around the `wait_for_visible` call then 🤔